src/modules/application/application.service.ts

```diff
@@ -33,11 +33,15 @@ export class ApplicationService {
    * returns single App with matching namespace
    * @param {String} owner
```

Comment:
Please adjust parameter documentation

Reply:
@JGiter We still need to describe the `withRoles` param. Is this description correct?
```suggestion
   * @param {String} owner
   * @param {object} [object.withRoles] - Include role data in query result
```